### PR TITLE
add get_pdb tool, fix rdkit_descriptors wrongly erroring

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: chemicaltoolbox/data_source/get_online_data/get_pdb.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<tool id="get_pdb" name="Get PDB file" version="0.1.0">
+    <description>from Protein Data Bank</description>
+    <requirements>
```
Comment (bgruening) [inline chemicaltoolbox/data_source/get_online_data/get_pdb.xml]: wget should be a dependency

File: chemicaltoolbox/data_source/get_online_data/get_pdb.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        wget https://files.rcsb.org/download/${pdb_id}.pdb -O '$output'
+    ]]></command>
+    <inputs>
+        <param name="pdb_id" type="text" label="PDB accession code"/>
```
Comment (bgruening) [inline chemicaltoolbox/data_source/get_online_data/get_pdb.xml]: Maybe we should only allow letters and digits here?

**Comment by bgruening** [comment]
Merci!

File: chemicaltoolbox/data_source/get_online_data/get_pdb.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<tool id="get_pdb" name="Get PDB file" version="0.1.0">
```
Comment (bgruening) [inline chemicaltoolbox/data_source/get_online_data/get_pdb.xml]: @simonbray a .shed.yml file was missing so this tool was not uploaded

Comment (simonbray) [reply chemicaltoolbox/data_source/get_online_data/get_pdb.xml]: whoops :/
https://github.com/bgruening/galaxytools/pull/862

**Pull request merged**